client/my-sites/upgrades/checkout/pay-button.jsx

```diff
@@ -113,15 +105,16 @@ var PayButton = React.createClass( {
 	},
 
 	completing: function() {
-		var text;
-		if ( hasFreeTrial( this.props.cart ) ) {
-			text = this.translate( 'Starting your free trial…', { context: 'Loading state on /checkout' } )
-		} else {
-			text = this.translate( 'Completing your purchase', { context: 'Loading state on /checkout' } )
+		if ( this.props.completingText ) {
+			return {
```

Comment:
Weird, I do have a space on my end. I think the github diff tool has some limitations. If you look here it does look fine https://github.com/Automattic/wp-calypso/blob/update/free-trial-checkout-page/client/my-sites/upgrades/checkout/pay-button.jsx#L109